protobuf/check_custom_value.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package Trento.Checks.V1;
+option go_package = "/;events";
+
+message CheckCustomValue {
+    string name = 1;
+    string value = 2;
```

Comment:
question: could be sending the current default value useful? I think it gives a better overview about what was changed

Reply:
Yes, that is an option. I don't have anything against that.
We are able to resolve the original value (not necessarily the default) and we have at hand the custom one, so yes not a big deal adding it.
Deal?